website/docs/integrations/datadog.md

```diff
@@ -14,9 +14,10 @@ Ensures that every setting change in ConfigCat is sent to DataDog as an Event.
 1. Have a <a href="https://www.datadoghq.com/" target="_blank">DataDog subscription.</a>
 2. Get a <a href="https://docs.datadoghq.com/account_management/api-app-keys/#api-keys" target="_blank">DataDog API Key.</a>
 ![datadog_event](/assets/datadog_apikey.png)
-1. Open the <a href="https://app.configcat.com/product/integrations" target="_blank">integrations tab</a> on ConfigCat Dashboard.
-2. Click on DataDog's CONNECT button and set your DataDog API key.
-3. You're all set. Go ahead and make some changes on your feature flags then check your Events in DataDog.
+3. Open the <a href="https://app.configcat.com/product/integrations" target="_blank">integrations tab</a> on ConfigCat Dashboard.
+4. Click on DataDog's CONNECT button and set your DataDog API key.
+5. OPTIONAL - Set the proper site of your datadog account. [More about Datadog site](https://docs.datadoghq.com/getting_started/site/).
```

Comment:
```suggestion
5. OPTIONAL - Set the proper site of your DataDog account. [More about Datadog site](https://docs.datadoghq.com/getting_started/site/).
```

Reply:
Right, although the correct name is `Datadog`, i updated on this page.